src/cli/cli.cpp

```diff
@@ -100,8 +100,6 @@ static OT_DEFINE_ALIGNED_VAR(sInterpreterRaw, sizeof(Interpreter), uint64_t);
 Interpreter::Interpreter(Instance *aInstance, otCliOutputCallback aCallback, void *aContext)
     : OutputImplementer(aCallback, aContext)
     , Output(aInstance, *this)
-    , mUserCommands(nullptr)
-    , mUserCommandsLength(0)
```

Comment:
Do `memset(&mUserCommands, 0, sizoef(mUserCommands))` from constructor?
So it all start as zero.

Reply:
Thank you, and I appreciate the noted change 👍